app/testonly/model.scala

```diff
@@ -47,6 +48,7 @@ object TestUser {
     utr                          = Utr.random(),
     hasSAEnrolment               = true,
     isOnIA                       = true,
+    nino                         = Some(Nino("AA000005A")),
```

Comment:
It would be nice to have random NINO number generator like we have for other bits.

Reply:
implemented